Docs 10035 develop compat matrix prototype oas3 pa--DON'T CLOSE (#94)
mulesoft/docs-general: modules/ROOT/pages/api-compatibility-matrix.adoc

= Anypoint Platform Compatibility Support for Restful-APIs
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> You can design an API in Anypoint Platform using the RESTful API Modeling Language (RAML) or the OpenAPI Specification (OAS) format based on your needs. Both RAML and OAS specifications implement a similar API model and provide the same features, except for the callback support in OAS 3.0. 

Reviewer: ```suggestion
You can design an API in Anypoint Platform using the RESTful API Modeling Language (RAML) or the OpenAPI Specification (OAS) format based on your needs. Both RAML and OAS specifications have a common REST API model and provide similar features. 
```
I wouldn't call out any specific differences here as it may be confusing. The tables outline all those differences already.

Author: Makes sense! Done! Thanks!

> RAML, which is optimized for design and reusability, implements _syntactic sugar_ to make descriptions smaller. Conversely, OAS enables you to simulate the components and libraries.

Reviewer: ```suggestion
```
I read the original sentence and there is no comparison (ie RAML vs OAS). I would remove this altogether as it doesn't provide any helpful information beyond what's said in the previous paragraph.

Author: Done! Thanks!

> MuleSoft API Tooling, which includes parser AMF and Language server (ALS), provides interoperability using common metadata model and enables data definition reuse, use of linting rules, and provides translation in both RAML and OAS specifications.

Reviewer: ```suggestion
MuleSoft's underlying metadata tools, which includes AMF: the parser, and ALS: the language server for IDEs, provides interoperability using a common metadata model and enables data definition reuse, use of linting rules, and provides translation to and from both RAML and OAS specifications.
```
Other options for `MuleSoft's underlying metadata tools`:
- `MuleSoft's underlying metadata libraries`
- `MuleSoft's underlying metadata components`

Author: Done! Thanks!

> //Jonathan--Why are we talking about API Tooling here suddenly? also, we need to give the users any best practice guidance here?

Reviewer: ```suggestion
```
It's not "API Tooling" per sé. Changed it.

Author: Done! Thanks!
